pkg/cmd/util/log/log.go

```diff
@@ -0,0 +1,190 @@
+// A logger implementation with signature compatible with "github.com/golang/glog", but
+// specifically targeted for pretty-printing for end users. Falls back to standard glog.
+// Loosely coupled to glog, some features are not implemented and some new ones were added.
+// Import as
+//   glog "github.com/openshift/origin/pkg/cmd/util/log"
+package log
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/golang/glog"
+	"github.com/openshift/origin/pkg/cmd/util"
+)
+
+const FATAL_EXIT_CODE = 255
+
+// Info logs to the INFO log.
+// Arguments are handled in the manner of fmt.Print; a newline is appended if missing.
+func Info(args ...interface{}) {
+	log(infoln, glog.Info, args...)
+}
+
+// Infoln logs to the INFO log.
+// Arguments are handled in the manner of fmt.Println; a newline is appended if missing.
+func Infoln(args ...interface{}) {
```

Comment:
Not necessarily, but it keeps the signature of everything in this pkg closer to what's in `github.com/golang/glog`, making it simpler to replace one with another.
On the other hand if we decide that's not an issue, a few funcs here could be removed.